File: crate/fixtures/documents/strings.rb
# A comment mentioning "the docs" is prose, not a literal.
=begin
A block comment mentioning "the manual".
=end

TEXT = <<~EOS
Dear reader,
Welcome.
EOS

RAW = <<-'PLAIN'
No #{interpolation} here.
  PLAIN

DOUBLE = "double quoted"
SINGLE = 'single quoted'
EMPTY = ''
queue = []
queue << item

File: crate/fixtures/documents/strings.pl
# A comment mentioning "the docs" is prose, not a literal.

=pod

A POD block mentioning "the manual".

=cut

my $heredoc = <<'END';
Dear reader,
Welcome.
END

my $double = "double quoted";
my $single = 'single quoted';
my $last = $#items;
my $empty = '';
